website/screens/common/HeaderCell.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+const HeaderCell = styled.th`
```

Comment:
Maybe include "Description" in the name to imply a more specialized use/scenario.

Reply:
Done